src/Api.php

```diff
@@ -160,6 +164,17 @@ function () use ($config) {
                     }
                 }
             );
+
+            $this->group(
+                Constants::ENDPOINT_TAXONOMY,
+                function () use ($config) {
+
+                    if ($config->taxonomies->post->enabled) {
+                        $this->post('/searches', TaxonomiesHandler::class . ':getTaxonomies')
```

Comment:
I made this same consideration and normally, I would agree. According to the REST conventions this would be a GET. However, in order for us to be able to expose a clearly documented API spec (Swagger) we should use a POST because the taxonomy keys are not known upfront and depend on the Grav configuration. This means that we cannot define the query parameters than can be used and therefore require a map of string keys and array values which can only be defined as a request body. This is why I explicitly use the /searches endpoint. This way I created a "ghost resource" called "searches" (plural) underneath the taxonomies resource to make it REST-compliant. This is a technique that I have used before in cases where advanced searching is required.

Reply:
Okay, that makes sense to me. Let's leave it as a POST request then. I hadn't considered more complex search cases, but you're right; it would be moreeeasily represented in the request body.